lib/kitchen_sink/math.ex

```diff
@@ -24,6 +30,9 @@ defmodule KitchenSink.Math do
   iex> ceil(29.49, 10)
   30
   """
+  def ceil(_, 0) do
```

Comment:
maybe we should have a default value for significance that makes this look like the standard ceil function